swagger.yaml

```diff
@@ -429,4 +469,53 @@ components:
           description: "Description2 of sub work stage"
         productCode:
           type: string
-          description: "Product code"
\ No newline at end of file
+          description: "Product code"
+
+      ProductSubWorkStage:
+        type: object
+        description: Information of product's sub workstage on a machine
+        required:
+          - id
+          - state
+          - machineId
+          - machineCode
+          - productCode
+          - productName
+          - startTime
+          - endTime
```

Comment:
Maybe remove this since we don't know is this required or not in lemonsoft?

Reply:
As far as I checked it's always present, but true, I will make it optional :+1: